package.json

```diff
@@ -89,6 +89,7 @@
     "hygen": "^6.2.0",
     "identity-obj-proxy": "^3.0.0",
     "jest": "^26.2.0",
+    "jest-environment-jsdom-sixteen": "^2.0.0",
```

Comment:
what's this doing? I don't see it it used

Reply:
It's in the top comment of the default.test.jsx:
```
/*
 Enzyme is using an old jsdom that has issues using waitFor on prototype
 methods and this component is using Component constructor and prototypes.
 This will set the proper jsdom environment for this specific test need
 until we can convert this away from the component model or update the test.
 @jest-environment jsdom-sixteen
*/
```
Basically, Enzyme is using a jsdom that has an issue with Class based components with async waitFor on prototype methods.  I have a ticket to update this removing the need for this module.

Reply:
gotcha no worries -- missed this, my bad